app/client/src/components/pages/WaterbodyReport.js

```diff
@@ -213,6 +213,30 @@ const introTextStyles = css`
 
 const listStyles = css`
   padding-bottom: 0;
+  &.document-list {
+    padding-left: 0;
+    width: 100%;
+  }
+`;
```

Comment:
```suggestion
const listStyles = css`
  padding-bottom: 0;
  padding-left: 0;
  width: 100%;
`;
```

The `listStyles` variable is only used once, so the `&.document-list` selector is not needed here.